### PR TITLE
Fix warnings about action name capitalization

### PR DESCRIPTION
**Pull request by Undin**

### CODE REVIEW

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -985,12 +985,12 @@
         </group>
         <action id="Rust.ToggleExternalLinterOnTheFlyAction"
                 class="org.rust.ide.actions.ToggleExternalLinterOnTheFlyAction"
-                text="Run external linter (cargo check / clippy) on the fly">
+                text="Run External Linter on the Fly">
```
Comment (mchernyavsky) [inline src/main/resources/META-INF/plugin.xml]: Without `cargo check / clippy` the option is not searchable via `Find Action`.

**Comment by Undin** [comment]
bors retry

**Pull request merged**